config/mbed_app.json

```diff
@@ -15,6 +15,7 @@
 			"bluenrg_ms.SPI_nCS": "BLE_SPI_NSS",
 			"bluenrg_ms.SPI_SCK": "BLE_SPI_SCK",
 			"cordio.desired-att-mtu": 251,
+			"cordio.rx-acl-buffer-size": 189,
```

Comment:
same here, it's not clear to me why the default value `70` was not working and why the value `189` is the one chosen.

my understanding (but I might be wrong) is that the value must be bigger than the `desired-att-mtu` size.

what would happen with a value of `256`?

Reply:
Never said it does not work
It is a size for a "packet" and it was downed to 189 to fit with iPad max buffer size (modulo some header or wrapper related to BLE protocol probably)
As you suggest with ChatGPT, value of 259 works